Assets/QA/Tests/XRDeviceActions/_SharedScripts/AxisControlActionStatus.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+using UnityEngine.Experimental.Input;
+using UnityEngine.Experimental.Input.Controls;
+using UnityEngine.UI;
+
+public class AxisControlActionStatus : MonoBehaviour {
+
+	public InputAction axisAction;
+
+    public Slider statusSlider;
+    public Text statusText;
+
+	// Use this for initialization
+	void Start () {
+        axisAction.Enable();
+        axisAction.performed += UpdateAxis;
+        axisAction.started += UpdateAxis;
+        axisAction.cancelled += UpdateAxis;
+	}
+
+    private void UpdateAxis(InputAction.CallbackContext context)
+    {
+        float value = ((AxisControl)(context.control)).ReadValue();
+        statusSlider.value = value;
+        //statusText.text = value.ToString();
```

Comment:
This should either be removed, or have a second comment explaining why it's temporarily removed (ideally with name/date).